projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -37,53 +35,50 @@
 
 ## 13.3-a.1 - 2024-03-12
 ### Enhancements
-- Display Email settings on Newsletter settings page [#36290]
-- Login block: Link to subscription management page with site URL as search query.
-- Member login block: link to individual subscription management page [#36200]
-- Pass API parameter to indicate when deleting a memberships product should cancel subscriptions [#35735]
+- Newsletters: Display Email settings on Newsletter settings page. [#36290]
 - Sharing: add a Threads sharing button and a Threads sharing button block. [#36220]
-- Social Menu & Social Media Icons: Add support for an SMS button. [#36176]
-- Uncheck users-new.php Invite user on load if WooCommerce plugin is activated [#36140]
-
-### Improved compatibility
-- Add metadata to the post to better diagnose need for Reader and Firehose [#36254]
+- Social: Add support for an SMS button. [#36176]
 
 ### Bug fixes
 - Sharing: avoid PHP warnings when using custom post types. [#36315]
 - Sharing: fix the display of the sharing block in some classic themes. [#36283]
 
 ### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
-- Add site migration endpoint [#36253]
-- Compile typescript extensions in Jetpack [#35904]
-- Contact Form: add deprecation warnings to Contact Form module codebase [#36040]
-- Contact Form: refactor editor to use forms package [#36089]
-- Contact Form: refactor endpoints to use forms package [#36087]
+- Add metadata to the post to better diagnose need for Reader and Firehose. [#36254]
+- Add site migration endpoint. [#36253]
+- Compile typescript extensions in Jetpack. [#35904]
```

Comment:
Is this something we usually include in changelogs? If so:

```suggestion
- Compile TypeScript extensions in Jetpack. [#35904]
```

Reply:
Yes, it's in the Other section which will not get published to the official readme.txt.